app/src/lang/translations/en-US.yaml

```
@@ -904,6 +905,7 @@ columns: Columns
 collection_setup: Collection Setup
 optional_system_fields: Optional Fields
 value_unique: Value has to be unique
+value_index: Value is indexed
```

Since strictly speaking it's not that value that is indexed, but the column, or in Directus speak, the field

```suggestion
value_index: Field is indexed
```